mep/people/views.py

```diff
@@ -308,14 +308,18 @@ def get_context_data(self, **kwargs):
 
         # address of the lending library itself; automatically available from
         # migration mep/people/migrations/0014_library_location.py
-        library = Location.objects.get(name='Shakespeare & Company')
-        context['library_address'] = {
-            'name': library.name,
-            'street_address': library.street_address,
-            'city': library.city,
-            'latitude': str(library.latitude),
-            'longitude': str(library.longitude),
-        }
+        try:
+            library = Location.objects.get(name='Shakespeare and Company')
+            context['library_address'] = {
```

Comment:
Am I remembering/understanding correctly that we have to set all this data in the view for easy hand off to display in leaflet?

Recommend moving it to a method or property on `Location` so we don't have this redundant code in the view.